constants/chatbotCommands.py

```diff
@@ -129,6 +129,22 @@ async def _help(fro: str, chan: str, message: list[str]) -> str:
     return "\n".join(l)
 
 
+@command(
+    trigger="!addbn",
+    syntax="<name>",
+    privs=privileges.ADMIN_MANAGE_PRIVILEGES,
+)
+async def addbn(fro: str, chan: str, message: list[str]) -> str:
+    """Add BN priveleges to a user"""
+    userid = await user_utils.get_id_from_username(fro)
+    if not (targetID := await user_utils.get_id_from_username(userid)):
+        return "Could not find user"
+    current_priveleges = await user_utils.get_privileges(userid)
+    new_priveleges = current_priveleges | 256
+
+    user_utils.set_privileges(userid, new_priveleges)
+
```

Comment:
we should return some user-friendly message here, like "cmyui has been made a beatmap nominator"

Reply:
since "cmyui" is variable depending on who is being made a bn, we can use a format string (or fstring) by prefixing a string with 'f' and using {x} syntax to denote variables

e.g.
```py
return f"{target_username} has been made a beatmap nominator"
```